pandas/tests/io/test_common.py

```diff
@@ -411,3 +414,298 @@ def test_is_fsspec_url():
     assert not icom.is_fsspec_url("random:pandas/somethingelse.com")
     assert not icom.is_fsspec_url("/local/path")
     assert not icom.is_fsspec_url("relative/local/path")
+
```

Comment:
can you make a new test file for all of these additions. test_user_agent.py or similar